crates/starknet-devnet/src/cli.rs

```diff
@@ -652,4 +695,20 @@ mod tests {
             ),
         }
     }
+
+    #[test]
+    fn check_if_method_with_incorrect_name_will_produce_an_error() {
```

Comment:
Why not add another test which lists some http paths or rpc methods, but succeeds?

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/34cb8ef811967625e9872d67eceba2179c627b65